cves/kernel/CVE-2016-4578.yml

```diff
@@ -313,9 +326,12 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: true
+  note: |
```

Comment:
You could reword this to something like, "There was discussion surrounding the possibility of tying this CVE to a similar one."